packages/edit-site/src/components/global-styles/hooks.js

```diff
@@ -19,9 +19,49 @@ import {
  * Internal dependencies
  */
 import { store as editSiteStore } from '../../store';
+import {
+	PRESET_METADATA,
+	getValueFromVariable,
+	getPresetVariableFromValue,
+} from '../editor/utils';
```

Comment:
Can we move these utils file to the`global-styles` folder ?

Reply:
Yes, we can move it there for now, but ideally, it should be part of the block editor as I'm proposing on https://github.com/WordPress/gutenberg/pull/34178.